src/testing_system/internal/adapter/assistants/ollama.py

```
@@ -21,17 +25,35 @@ def _ensure_model(self) -> None:
             response = requests.get(f"{self.base_url}/api/tags", timeout=10)
         except requests.ConnectionError:
             raise RuntimeError(f"Ollama not available at {self.base_url}")
-        
+        logger.debug(f"responce is recieved from ollama: {response}")
         models = response.json().get("models", [])
         model_names = [m["name"] for m in models]
         
         if self.model not in model_names:
-            print(f"Downloading model {self.model}...")
+            logger.info(f"Downloading model {self.model}...")
             response = requests.post(
```

Ruff (0.15.13)

[error] Probable use of `requests` call without timeout

(S113)